src/wp-includes/functions.php

```diff
@@ -1904,7 +1904,8 @@ function wp_nonce_field( $action = -1, $name = '_wpnonce', $referer = true, $ech
  * @return string Referer field HTML markup.
  */
 function wp_referer_field( $echo = true ) {
-	$referer_field = '<input type="hidden" name="_wp_http_referer" value="' . esc_attr( wp_unslash( $_SERVER['REQUEST_URI'] ) ) . '" />';
+	$request_url   = remove_query_arg( '_wp_http_referer', wp_unslash( $_SERVER['REQUEST_URI'] ) );
```

Comment:
```suggestion
	$request_url   = remove_query_arg( '_wp_http_referer' ) );
```

Reply:
Any reason to specifically add the current URL?  Underneath, it defaults to that anyway if the second param isn't passed.

Reply:
Agreed with @dream-encode. Don't needs to add current URL.